docs/release.sh

```diff
@@ -116,17 +84,20 @@ invalidate_cache() {
 
 	aws configure set preview.cloudfront true
 
-	files=$(find site/ -not -name "*.md*" -type f | sed 's/site\///g')
+	# Get all the files
+	# not .md~ files
+	# not files w spaces bc they mess up the array
```

Comment:
Also, if you want to avoid the `sed`, you can do `files=( $(cd site && find -not ....) )`.